.travis.yml

```diff
@@ -27,7 +27,7 @@ env:
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
         - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='pytest>=3'
```

Comment:
Note: I tried several ways (e.g., upping minversion in setup.cfg, setting PYTEST_VERSION in .travis.yml) but this is the only way that works for me. Otherwise, Travis mixes 2.7 or 3.0.5 for different jobs in the same build however it likes.